modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -203,47 +203,73 @@ Each provider has its own configuration map that can be passed to the `genai.vec
 | model
 | STRING
 | The name of the model you want to invoke. +
-Supported values: `"textembedding-gecko@001"`
+ +
+Supported values: +
+ +
+
+label:new[Introduced in 5.17]:
+
+* `"textembedding-gecko@001"`
+
+label:new[Introduced in 5.19]:
+
+* `"textembedding-gecko@002"`
+* `"textembedding-gecko@003"`
+* `"textembedding-gecko-multilingual@001"`
```

Comment:
agree with @parnmatt  - this is how we usually would render it.
```suggestion

* `"textembedding-gecko@001"` label:new[Introduced in 5.17]
* `"textembedding-gecko@002"` label:new[Introduced in 5.19]
* `"textembedding-gecko@003"` label:new[Introduced in 5.19]
* `"textembedding-gecko-multilingual@001"` label:new[Introduced in 5.19]
```

Reply:
That just seemed very verbose but if you prefer that I'm all for it 👍